src/commands/pulls/update.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @see: https://octokit.github.io/rest.js/#api-Pulls-update
+ * const result = await octokit.pulls.update({owner, repo, number, title, body, state, base, maintainer_can_modify})
+ * /repos/:owner/:repo/pulls/:number
+ */
+const flow = require("lodash.flow")
+const fs = require("fs")
+
+const commonYargs = require("../../../lib/common-yargs")
+const printOutput = require("../../../lib/print-output")
+const authenticatedOctokit = require("../../../lib/octokit")
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([
+		commonYargs.withToken,
+		commonYargs.withJson,
+		commonYargs.withBase,
+		commonYargs.withOwner,
+		commonYargs.withRepo,
+		commonYargs.withNumber,
+		commonYargs.withBody,
+		commonYargs.withTitle,
+	])
+
+	return baseOptions(yargs)
+		.option("state", {
+			describe: "State of this Pull Request. Either open or closed. Allowed values: open, closed",
+			type: "string",
+		})
+		.option("maintainer_can_modify", {
+			describe: "Indicates whether maintainers can modify the pull request.",
+			type: "string",
+		})
+}
+
+/**
+ * Return the contents of a pull request body and update a pull request.
+ *
+ * @param {object} argv - argv parsed and filtered by yargs
+ * @param {string} argv.token
+ * @param {string} argv.json
+ * @param {string} argv.owner
+ * @param {string} argv.repo
+ * @param {string} argv.number
+ * @param {string} argv.title
+ * @param {string} argv.branch
+ * @param {string} [argv.base]
+ * @param {string} [argv.body]
+ * @throws {Error} - Throws an error if any required properties are invalid
+ */
+const handler = async ({ token, json, base, body, owner, repo, number, title, branch }) => {
+
+	// Ensure that all required properties have values
+	const requiredProperties = {
+		owner,
+		repo,
+		number,
+		title,
```

Comment:
Title is a required property when updating? Seems not right.

Reply:
You're right -- title 's optional for `update`s